src/libs/test-utils.js

```diff
@@ -1,6 +0,0 @@
-import * as Utils from 'src/libs/utils'
```

Comment:
I was looking for a test file related to utils and ran across this… which is unused.